bootstrap/ic-user-projects/create-projects-and-resources.bash

```diff
@@ -283,7 +283,7 @@ spec:
   resources:
     requests:
       storage: 5Gi
-  storageClassName: ocs-storagecluster-ceph-rbd
+  storageClassName: ocs-external-storagecluster-ceph-rbd
```

Comment:
Same as prior comment .. simply delete and let the default storageclass get used